README.md

```diff
@@ -66,22 +66,22 @@ artifact_dir: "/app/beetmoverworker/artifact"
 task_log_dir: "/app/beetmoverworker/artifact/public/logs"
 
 # chainoftrust config
-sign_chain_of_trust: false
-verify_chain_of_trust: false
-verify_cot_signature: false
+sign_chain_of_trust: true
+verify_chain_of_trust: true
+verify_cot_signature: true
 cot_job_type: beetmover
 
 # gpg homedir config
 ## only used when verify_chain_of_trust is true
-base_gpg_home_dir: "/app/beetmoverworker/gpg"
-gpg_lockfile: "/app/beetmoverworker/gpg_homedir.lock"
-git_key_repo_dir: "/app/beetmoverworker/key_repo"
-git_commit_signing_pubkey_dir: "/app/beetmoverworker/valid_git_fingerprints/"
-last_good_git_revision_file: "/app/beetmoverworker/git_revision"
-pubkey_path: "/app/beetmoverworker/my_pubkey.asc"
-privkey_path: "/app/beetmoverworker/my_privkey.asc"
+base_gpg_home_dir: "/app/beetmoverworker-1.0.0b4/gpg"                                                                                 [48/9992]
+gpg_lockfile: "/app/beetmoverworker-1.0.0b4/gpg_homedir.lock"
+git_key_repo_dir: "/app/beetmoverworker-1.0.0b4/key_repo"
+git_commit_signing_pubkey_dir: "/app/beetmoverworker-1.0.0b4/valid_git_fingerprints/"
+last_good_git_revision_file: "/app/beetmoverworker-1.0.0b4/git_revision"
+pubkey_path: "/app/beetmoverworker-1.0.0b4/my_pubkey.asc"
+privkey_path: "/app/beetmoverworker-1.0.0b4/my_privkey.asc"
```

Comment:
yeah, this was added late to the commit and I forgot to strip the 1.0.0b4 from the paths. I have a few beetmover versions locally. Will fix.